### ISSUE TITLE
nidobata init すると「uninitialized constant GraphQL::Client::HashWithIndifferentAccess::Forwardable (NameError)」が発生します

### ISSUE THREAD

**Issue by ghost** [idobata/nidobata#7]
nidobata init すると 「uninitialized constant GraphQL::Client::HashWithIndifferentAccess::Forwardable (NameError)」が発生しました。

## 環境

- ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-darwin16]
- nidobata-0.6.0

## ログ

```console
~/Desktop %nidobata init
/Users/naiad/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/graphql-client-0.11.2/lib/graphql/client/hash_with_indifferent_access.rb:11:in `<class:HashWithIndifferentAccess>': uninitialized constant GraphQL::Client::HashWithIndifferentAccess::Forwardable (NameError)
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/graphql-client-0.11.2/lib/graphql/client/hash_with_indifferent_access.rb:10:in `<class:Client>'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/graphql-client-0.11.2/lib/graphql/client/hash_with_indifferent_access.rb:5:in `<module:GraphQL>'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/graphql-client-0.11.2/lib/graphql/client/hash_with_indifferent_access.rb:4:in `<top (required)>'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/graphql-client-0.11.2/lib/graphql/client/errors.rb:2:in `<top (required)>'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/graphql-client-0.11.2/lib/graphql/client/schema/object_type.rb:5:in `<top (required)>'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/graphql-client-0.11.2/lib/graphql/client/definition.rb:5:in `<top (required)>'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/graphql-client-0.11.2/lib/graphql/client.rb:7:in `<top (required)>'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/nidobata-0.6.0/lib/nidobata.rb:9:in `<top (required)>'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /Users/naiad/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/nidobata-0.6.0/exe/nidobata:3:in `<top (required)>'
	from /Users/naiad/.rbenv/versions/2.4.1/bin/nidobata:22:in `load'
	from /Users/naiad/.rbenv/versions/2.4.1/bin/nidobata:22:in `<main>'
```

**Comment by takkanm** [comment]
It's graphql-client Bug.
https://github.com/github/graphql-client/pull/111

If you use nidobata without bundler, you should use v0.5.0.

**Comment by ghost** [comment]
Thank you for reply. I understand the cause.

**Comment by hibariya** [comment]
@taki3 Thank you for reporting the issue 😺  I've yanked nidobata-0.6.0 (takkanm suggested it on [idobata/oss](https://idobata.io/#/organization/idobata/room/oss)). We'll release the fixed version (0.6.1) when the next graphql-client is released.
